numpy/core/src/common/simd/vsx/math.h

```diff
@@ -33,4 +33,10 @@ NPY_FINLINE npyv_f32 npyv_square_f32(npyv_f32 a)
 NPY_FINLINE npyv_f64 npyv_square_f64(npyv_f64 a)
 { return vec_mul(a, a); }
 
+// Maximum, natively mapping with no guarantees to handle NaN.
+#define npyv_max_f32 vec_max
+#define npyv_max_f64 vec_max
+#define npyv_maxp_f32 vec_max
```

Comment:
`vec_max` supports IEEE

Reply:
Let me append the description for maxp just the same as other simd files.